lib/index.js

```diff
@@ -39,7 +39,13 @@ internals.implementation = function (server, options) {
       // otherwise use the same key (String) to validate all JWTs
       var decoded;
       try {
-        decoded = JWT.decode(token); // decode is non-io and fast enough to not have to be async
+		if (options.completeToken === true) { 
```

Comment:
@nelsonic look at https://github.com/auth0/node-jsonwebtoken/blob/master/index.js#L31-L38 to see what the _complete_ option does. You can see that without this option, the JWT payload _content_ is returned. With _complete_ set to true, _payload_ is returned as one property (`payload:`) of the returned object.
Current users accessing a JWT payload property like `decoded.sub` would have to update their code to use `decoded.payload.sub` if _complete_ was always set to true.
This is the reason why always setting this to true would break backward compatibility.